client/cypress/support/shoppinglist.po.ts

```diff
@@ -39,13 +39,33 @@ export class ShoppinglistPage {
     return cy.get('.store-tabs-group .mat-tab-body[role="tabpanel"]').eq(position);
   }
 
-  getStoreItems(position: number) {
-    this.getStoreProductsPanel(position);
+  /**
+   * Get all the items in the shopping list for the store at the
+   * specified position.
+   *
+   * @param storePosition The position in the tab group of the desired
+   * store; the left-most store is at position zero.
+   * @returns the list of all the shopping list items for that store
+   */
+  getStoreItems(storePosition: number) {
+    // Does this next line actually _do_ anything? We don't capture
+    // the return value or act on it, so if it does do anything, it's
+    // entirely via side-effects, which at a minimum means that the
+    // name of `getStoreProductsPanel()` is dodgy.
+    this.getStoreProductsPanel(storePosition);
```

Comment:
I'm genuinely unsure what's happening here. I'm a little suspicious that this line does nothing, and things "work" because we're only removing an item from the first (left most) store. I'll have to explore that further as I work on this.